src/modules/categories/selectors/categories.js

```diff
@@ -8,9 +8,20 @@ export default function () {
 
 export const selectCategories = createSelector(
   selectCategoriesState,
-  categories => Object.keys(categories || {})
-    .map(category => ({ popularity: categories[category].popularity, category: categories[category].category }))
-    .sort(popularityDifference),
+  (categories) => {
```

Comment:
FWIW this can be rewritten as
```
export const selectCategories = createSelector(
  selectCategoriesState,
  categories =>
    Object.values(categories)
      .filter(c => c && !!c.category)
      .map(({ category, popularity }) => ({ category, popularity }))
      .sort(popularityDifference),
)
```